core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/parsing/DataTypeClassNameParser.java

```diff
@@ -60,6 +61,30 @@ public DataType parse(
       String toParse,
       Map<CqlIdentifier, UserDefinedType> userTypes,
       InternalDriverContext context) {
+    // We take keyspaceId as a parameter because of the parent interface, but it's actually unused
+    // by this implementation.
+    return parse(toParse, userTypes, context, context.getSessionName());
+  }
+
+  /**
+   * Simplified parse method for external use.
+   *
+   * <p>Note that this method is intended for use in Cassandra's UDF implementation (the current
+   * version uses the similar method from driver 3).
+   */
+  public DataType parse(String toParse, AttachmentPoint attachmentPoint) {
```

Comment:
is the Cassandra-core implementing `AttachmentPoint` and provides own DriverContext?
Maybe we can create a simple unit test for that public method?

Reply:
We'll have to make one, it will just hold the DSE Registry and the most recent Driver version